tests/integration/integration.go

```diff
@@ -155,17 +168,18 @@ func setInstances() {
 		toEngine := make(chan common.Message, 1)
 		db := memdb.New()
 
-		v := createVM(
-			vm.WithManualGossiper(),
-			vm.WithManualBuilder(),
+		v, err := createVM(
+			vm.NewOption("manualGossiper", vm.WithManualGossiper()),
+			vm.NewOption("manualBuilder", vm.WithManualBuilder()),
```

Comment:
can we move both of the names into `WithManualGossiper` directly, so that each service is responsible for providing its own namespace.